baggage/templates/baggage_add.html

```diff
@@ -0,0 +1,139 @@
+{% extends "base_tabs.html" %}
+
+{% load static %}
+{% load bootstrap3 %}
+{% load baggage_tags %}
+
+{% block extra_head %}
+    <link rel="stylesheet" href="{% static 'css/baggage.css' %}">
+    <script src="{% static 'lib/instascan.min.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/baggage_add.js' %}"></script>
+    <script src="{% static 'lib/typeahead.min.js' %}"></script>
+    <script src="{% static 'js/form_modifiers.js' %}"></script>
+{% endblock %}
+
+{% block head_title %}Baggage check-in{% endblock %}
+
+{% block panel %}
+    <dl class="dl-horizontal" style="margin-bottom: 0;">
+	{% include 'include/field.html' with desc='Name' value=user.name %}
+	{% include 'include/field.html' with desc='Email' value=user.email %}
+	{% include 'include/field.html' with desc='Position' value=position %}
+    </dl>
+    
+    
+    <script>
+        $(document).ready(function(){
+            make_field_typeahead('bag_type', '{% static 'types.json' %}');
+            make_field_typeahead('bag_color', '{% static 'colors.json' %}');
+        })
+    </script>
+{% endblock %}
+
+{% block cols-panel %}col-md-6 col-md-offset-3{% endblock %}
+
+{% block out_panel %}
+    <form id="baggage-form" action="" method="post" enctype="multipart/form-data">
+	{% csrf_token %}
+	<div class="form-group">
+	    <label class="control-label" for="bag_type">Type</label>
+	    <input type="text" id="id_bag_type" class="form-control" name="bag_type" autocomplete="off" />
+	</div>
+	<div class="form-group">
+	    <label class="control-label" for="bag_color">Color</label>
+	    <input type="text" id="id_bag_color" class="form-control" name="bag_color" autocomplete="off" />
+	</div>
+	<div class="form-group">
+	    <label class="control-label" for="baggage-scan-image">Image</label>
+	    <div id="baggage-scan-image"></div>
```

Comment:
Any chance to add a button to freeze the picture to avoid taking accidental pictures of myself? 🙃

Reply:
I will add something to enable/disable that, in #53.